Makefile

```diff
@@ -35,7 +35,7 @@ DIST_FILE=aci-containers.tgz
 
 DOCKER_HUB_ID ?= noiro
 DOCKER_TAG ?=
-BUILD_CMD ?= go build -v -trimpath
+BUILD_CMD ?= go build -v -trimpath -race
```

Comment:
before merging we would need to revert this and L50

Reply:
sure, testing is in process